core/chains/evm/config/toml/config.go

```diff
@@ -399,6 +399,11 @@ func (c *Chain) ValidateConfig() (err error) {
 		err = multierr.Append(err, commonconfig.ErrInvalid{Name: "MinIncomingConfirmations", Value: *c.MinIncomingConfirmations,
 			Msg: "must be greater than or equal to 1"})
 	}
+
+	if c.NodePool.FinalizedBlockPollInterval.Duration() > 0 && !(*c.FinalityTagEnabled) {
```

Comment:
Does this imply that we could have `FinalityTagEnabled` enabled but not poll for the finalized block? I know we're not using `stateLatestFinalizedBlockNumber` for health just yet, but when we do, we'd always want polling enabled if finality tag is enabled right?

Reply:
Yes, we need to reverse check instead.
If FinalityTagEnabled is true, then FinalizedBlockPollInterval must be non-zero.
If FinalityTagEnabled is false, then FinalizedBlockPollInterval gets ignored.

Reply:
Why don't we just simplify that and track latest finalized block always? The only difference is that when `FinalityTagEnabled` is true then we should ask chain about finality, otherwise latest finalized block is just head - FinalityDepth

Reply:
I was not sure if we want it enabled be default. 
I'll add fallback value and remove the check